src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void inputChecker() {
+        Scanner in = new Scanner(System.in); // Declared a scanner object
+        String line; // Declared a string object to take in user input
+        Task[] list; // Declared a list with string data type
+        list = new Task[100]; // created a list of 100 elements
+        int counter = 0;
+        int index;
+        Task t;
+
+        // Start of user input
+        while (true) {
+            line = in.nextLine(); // Takes in user input
+
+            boolean mark = line.trim().toLowerCase().startsWith("mark");
```

Comment:
For boolean values, consider using a prefix such as is, has, was in front of "mark" and "unmark"